test/profile.test.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
Just use mock and handlers in the tests folder. it will be easier as you will be able to actual data